### ISSUE TITLE
Consistency Issue

### ISSUE THREAD

**Issue by kryztoval** [CraterCrash/godot-orchestrator#846]
### Describe the bug

A "Get Parent" from a Node returns Null or an empty string on first call.

I can't tell what is going on but it almost feels as if the instruction is not able to finish before the flow of the program ends.

### Expected behavior

Get Parent should return the parent node

### Actual behavior

Get Parent returns an empty string or a null.

### How to Reproduce?

- Scene : Block
- - Scene : Cell
- - - Button : CellButton { button_up() -> On Cell Button Up Event }

- Scene : pad
-  -  Button : key { button_up() -> On Button Up Event }

pad.torch: On Button Up Event -> Push Warning ( Get Parent ( Get Self ) )

cell.torch: On Cell Button Up Event -> InstantiateScene ( res://pad.tscn ) -> AddChild( [self], Static Root)

--- Run ---
Click on the cell button - Wait for the pad scene to be displayed
Click on the key button
The message will read something like "pad.torch:10 @ _on_button_up():" but no parent node will be displayed.
Click the key button again and now the name is properly displayed.

if you do either of these two changes it will work correctly:
- pad.torch: On Button Up Event -> Push Warning ( Get Node Name (Get Parent ( Get Self ) ) )
- cell.torch: On Cell Button Up Event -> Push Warning () -> InstantiateScene ( res://pad.tscn ) -> AddChild( [self], Static Root)

### Godot full version

v4.4.dev7.official [46c8f8c5c]

### Orchestrator version

v2.2.dev1.official [a09d0bda6]

**Comment by Naros** [comment]
I wonder if this is an issue with Godot 4.4, what if you use 4.3 @kryztoval ?

**Comment by kryztoval** [comment]
As requested, results (version, result for first click, result for second click)
Godot 4.3, empty, empty
Godot 4.4dev7, empty, Object Name <class name: id>
Godot 4.4beta2, empty, empty

It seems really weird that the inconsistency seems to be exclusive to 4.4dev7. I was expecting it to always return the Object name and Id, but if it never returns it then it should also be fine, as long as it is consistent!

I can only guess why in version 4.4dev7 it somehow returned a different auto conversion to string path.

Workout path 1 works in all versions
workout path 2 only works in 4.4dev7, all other versions still return a null/empty string.

This doesn't seem to be an orcestrator bug but rather a Godot change. I wonder why the auto Name<Class:id> auto toString was added then removed, and why it worked after push warning was called at least once.

**Comment by Naros** [comment]
That doesn't surprise me at all. The `devX` builds of the Godot editor are often in flux, and each PR merged is tested on its face in merit & often not widely across the entire editor ecosystem. Once the team is ready to focus on feature freeze and beta builds, they begin rigorous regression testing, identifying these issues and fixing them.

This is generally why we rarely begin any sort of support for new Godot builds until at least the `beta1` or `beta2` builds, and why we never commit to any official support for the new Godot build until it reaches `stable`.

**Comment by kryztoval** [comment]
Yeah, I think we can close this issue as a non issue. I will keep this in mind next time I am using a dev build of Godot.

Thank you :)
